model-optimizer/mo/graph/graph.py

```diff
@@ -1097,6 +1101,27 @@ def rename_nodes(nodes: List[tuple]):
     for node, name in nodes:
         rename_node(node, name)
 
+
+def get_attribute_between_nodes(node1: Node, node2: Node, attr_name: str):
+    for edge_idx in node1.out_edges():
+        edge = node1.out_edge(edge_idx)
+        out_port = edge['out']
+        out_node = node1.out_node(out_port)
+        if out_node.soft_get('id') is node2.soft_get('id'):
```

Comment:
the `id` attribute always exist in the node so there is no need to check it with `soft_get` function.

So, please, change this line to `if out_node.id == node2.id` (use `==` instead of `is`).

The same comment for the function below.